tap/api/api.go

```diff
@@ -132,11 +133,16 @@ type MizuEntry struct {
 	EstimatedSizeBytes      int    `json:"-" gorm:"column:estimatedSizeBytes"`
 }
 
+type FullEntryWithPolicy struct {
+	RulesMatched []map[string]interface{} `json:"rulesMatched,omitempty"`
+	Entry        MizuEntry            `json:"entry"`
+}
+
 type MizuEntryWrapper struct {
 	Protocol       Protocol  `json:"protocol"`
 	Representation string    `json:"representation"`
 	BodySize       int64     `json:"bodySize"`
-	Data           MizuEntry `json:"data"`
+	Data           FullEntryWithPolicy `json:"data"`
```

Comment:
```suggestion
	Data MizuEntry `json:"data"`
	RulesMatched []map[string]interface{} `json:"rulesMatched,omitempty"`	
```

Reply:
I think it will be better to put the rules near the entry and not wrap it in single item because it creates thenested object. so in frontend you have it like `data.entry.entry` 
wdyt?